add bitnami compat package for flux-helm-controller

sanjayk0508 opened the pull request:
Added bitnami compat package for flux-helm-controller, following the [Dockerfile](https://github.com/bitnami/containers/blob/main/bitnami/fluxcd-helm-controller/1/debian-12/Dockerfile)

### Pre-review Checklist

<!--
This checklist is mostly useful as a reminder of small things that can easily be
forgotten – it is meant as a helpful tool rather than hoops to jump through.

At the moment of this PR you have the most information on what all the change
will affect, so please take the time to jot it down.

Put an `x` in all the items that apply, make notes next to any that haven't been
addressed, and remove any items that are not relevant to this PR.

-->

#### For new package PRs only
<!-- remove if unrelated -->
- [x] This PR is marked as fixing a pre-existing package request bug
  - [x] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [ ] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [ ] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams
<!-- remove if unrelated -->
- [ ] The upstream project actually supports multiple concurrent versions.
- [x] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)
- [ ] If non-streamed package names no longer built, open PR to withdraw them (see [WITHDRAWING PACKAGES](https://github.com/wolfi-dev/os/blob/main/WITHDRAWING_PACKAGES.md))

#### For package updates (renames) in the base images
<!-- remove if unrelated -->
When updating packages part of base images (i.e. cgr.dev/chainguard/wolfi-base or ghcr.io/wolfi-dev/sdk)
- [ ] REQUIRED cgr.dev/chainguard/wolfi-base and ghcr.io/wolfi-dev/sdk images successfully build
- [ ] REQUIRED cgr.dev/chainguard/wolfi-base and ghcr.io/wolfi-dev/sdk contain no obsolete (no longer built) packages
- [ ] Upon launch, does `apk upgrade --latest` successfully upgrades packages or performs no actions

#### For security-related PRs
<!-- remove if unrelated -->
- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs
<!-- remove if unrelated -->
- [ ] The `epoch` field is reset to 0

#### For PRs that add patches
<!-- remove if unrelated -->
- [ ] Patch source is documented

sanjayk0508 committed:
add bitnami-compat

Signed-off-by: Sanjay Kumar <sanjay.kumar@chainguard.dev>

Dentrax commented on flux-helm-controller.yaml:
```diff
@@ -33,6 +33,25 @@ pipeline:
+          cp LICENSE ${{targets.subpkgdir}}/opt/bitnami/licenses/LICENSE
+    test:
+      pipeline:
+        - runs: |
```
Can we have `test/daemon-check-output` test here? Example: https://github.com/wolfi-dev/os/blob/45214d8b0fd5427711f02174c01a3bc1c175b4ce/repmgr-17.yaml#L184-L201

sanjayk0508 replied:
tried digging in and experimenting, but it seems we can't use `test/daemon-check-output` because of the flux binary. Please let me know if you think it's doable or possible.

Dentrax replied:
Moved discussion to here: https://chainguard-dev.slack.com/archives/C0627BW5GLF/p1741267781421389

kranurag7 replied:
thank you for the suggestion here, added test/daemon-check-output via https://github.com/wolfi-dev/os/pull/44300/commits/b907513bd1c5d0eb4cbf31b0dc4847a02c317bb3

Dentrax replied:
Thanks! The tests does seem good but any possibility if we do call bitnami's entrypoint with required envs? (If possible with the `accounts` set): https://github.com/bitnami/containers/blob/f14bcfa9081f0e26e18ee3c612025f08ba7c8288/bitnami/fluxcd-helm-controller/1/debian-12/Dockerfile#L48-L53

Dentrax replied:
We're not using `entrypoint.sh` so this is good to go!

kranurag7 committed:
remove bitnami/comapt pipline and use test/daemon-check-output

removing bitnami/compat given entrypoint is a go binary.
remove build time deps given go/build pulls in the require deps.

uses test/daemon-check-output in test pipeline.
Signed-off-by: kranurag7 <81210977+kranurag7@users.noreply.github.com>

kranurag7 committed:
update subpackge name

Signed-off-by: kranurag7 <81210977+kranurag7@users.noreply.github.com>

kranurag7 committed:
epoch bump 

Signed-off-by: kranurag7 <81210977+kranurag7@users.noreply.github.com>

kranurag7 committed:
use account in testing

Signed-off-by: kranurag7 <81210977+kranurag7@users.noreply.github.com>

kranurag7 committed:
Revert "use account in testing"

This reverts commit 1f93dfd59acd8811a1e1bfe132e303f102641851.
